### DISCUSSION TITLE
Are BLASTx annotations interchangeable?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1145] [Q&A] [answered]
I'm prepping my new hemat_transcriptomev1.6 analysis for GO-MWU, and part of it involves cross-referencing with a BLASTx annotation table to obtain accession IDs.

When analyzing cbai_transcriptomev2.0 (which has both hematodinium and Chionoecetes sequences), I used [this table](https://gannet.fish.washington.edu/Atumefaciens/20200608_cbai_diamond_blastx_v2.1_v3.1/cbai_transcriptome_v2.1.blastx.outfmt6), available [from Genomic Resources](https://robertslab.github.io/resources/Genomic-Resources/)

I believe (though am not 100% sure) that hemat_transcriptomev1.6 is derived from cbai_transcriptomev2.0. Is that the case? And if so, can I use the previously-used table to cross-reference sequence IDs and accession IDs?

**Comment by sr320** [comment, marked as answer]
Might it be best to generate a new blastx annotation table...

**Reply by aspencoyle** [reply]
Sounds good! Do I just BLAST my transcriptome against a database (say, the UniProt database) and then merge in R?

Roughly following [this workflow here](https://adnguyen.github.io/2017_Ecological_Genomics/Tutorial/2017-04-05_annotation_tutorial3.html)

**Reply by kubu4** [reply]
> hemat_transcriptomev1.6 is derived from cbai_transcriptomev2.0. Is that the case?

No.

Deets can be found on Genomic Resources docs.

**Reply by sr320** [reply]
@afcoyle Yes - blast to Swiss-Prot/UniProt
